harvest/api/paper.py

```diff
@@ -305,4 +331,36 @@ def order_option_limit(self, side: str, symbol: str, quantity: float, limit_pric
         }
         return ret
 
+    # ------------- Helper methods ------------- #
+
+    def _calc_equity(self):
+        """
+        Calculates the total worth of the broker by adding together the 
+        worth of all stocks, cryptos, options and cash in the broker.
+        """
+        e = 0
+        for asset in self.stocks + self.cryptos + self.options:
+            add = asset['avg_price'] * asset['quantity']
+            if 'multiplier' in asset:
+                add = add * asset['multiplier']
+            e += add
+        e += self.cash
+        return e
 
+    def apply_commission(self, inital_price: float, commission_fee, side: str) -> float:
```

Comment:
applies a commission, the returned value is how much the user actually has to pay or actually receives after the commission is applied